pkg/broker/registry.go

```diff
@@ -146,6 +154,14 @@ func (svc *BrokerService) IsEnabled() bool {
 	return viper.GetBool(svc.EnabledProperty())
 }
 
+// IsRoleWhitelistEnabled returns false if the operator has explicitly disabled
+// the role whitelist for this service or true otherwise.
+func (svc *BrokerService) IsRoleWhitelistEnabled() bool {
+	whitelistExists := len(svc.ServiceAccountRoleWhitelist) > 0
+	userEnabled := viper.GetBool(svc.RoleWhitelistProperty())
```

Comment:
It is possible, because the property name/environment variable mappings are implicit (`(s/-./_/)`) computed anyone not using the form to configure the software could set the environment variable to true for any service.

Some of the plans don't have whitelists because role is the "service", like the Stackdriver or Datastore ones so the roles are hard-coded.

We could instead let the operator enter a (CSV?) list of roles with the specified roles as default at the risk they do something wrong, but the benefit being they have more control. Being that environment variables don't have nil values, them entering 0 roles would be interpreted as choosing the default which might be confusing in the web UI. The idea of user-defined list of roles is growing on me. Thoughts?